examples/set-a-custom-referrer/README.md

```diff
@@ -0,0 +1,27 @@
+# Set a Custom Referrer
+
+**Test Code**: [index.js](index.js)
+
+**Tested Page**: [TestCafe Example Page](https://devexpress.github.io/testcafe/example/)
+
+The [HTTP referrer](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referer) (otherwise known as referer) is a request header that lets a webpage know where the user has navigated from.
+
+This example shows how to set a custom referrer during tests.
```

Comment:
`This example shows how to set a custom referrer for the specified requests during tests.`